stixpy/calibration/visibility.py

```diff
@@ -286,21 +302,20 @@ def create_visibility(meta_pixels):
     amplitude_error = np.sqrt((real / observed_amplitude * real_err) ** 2 + (imag / observed_amplitude * imag_err) ** 2)
 
     # Apply pixel correction
-    phase += 46.1 * u.deg  # Center of large pixel in terms morie pattern phase
-    # TODO add correction for small pixel
+    pix_set = meta_pixels["pixels"]
+    if pix_set in {"big", "top", "bottom"}:
```

Comment:
So I think the combination would be: 
all top+small bottom+small big+small = 45,
top, bottom or big = 46.1,
small = 22.5

But I think we should match IDL as best as possible for the moment so see https://github.com/i4Ds/STIX-GSW/blob/b2bc64b2543687aa8700e3fadbc97eb325bc1673/stix/idl/processing/vis/stx_pixel_data_summed2visibility.pro#L128-L147

Reply:
So you mean we should not implement `'top/bottom+small'`.

`'big+small'` is covered by `'all'`.